test/gie/builtins.gie

```diff
@@ -2212,6 +2288,30 @@ expect  -0.001790494 0.000895247
 accept  -200 -100
 expect  -0.001790496 -0.000895247
 
+accept -140.0 22.0
```

Comment:
same remark as above: those tests points aren't super relevant

Reply:
I added in the roundtrip option for all zones in my latest commit, but I am seeing failures with infs in zones 11 and 12. I am stumped. Do you see anything obvious with how I implemented the tests?

Reply:
Choosing slightly different values eliminates the inf issue.  Could it be because I chose values that lie on the true longitude for the Mollewiede projections?

Reply:
```
     FAILURE in builtins.gie(2283):
     roundtrip deviation: inf mm, expected: 0.100000 mm
```

Reply:
> Choosing slightly different values eliminates the inf issue. Could it be because I chose values that lie on the true longitude for the Mollewiede projections?

Spoke too soon on this. It doesn’t solve the problem.

Reply:
This ended up being a typo in the forward projection.  Fixed in 4dc21af.